deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable node/no-missing-require */
```

Comment:
I think the plugin just doesn't understand the local dependencies like `"@yari-internal/constants": "file:../../../libs/constants",` in the package.json, so ignored at the file level here. A few others under the `libs/` so maybe a better solution is needed globally later